helm_deploy/values-preprod.yaml

```diff
@@ -10,18 +10,20 @@ generic-service:
     host: electronic-monitoring-datastore-api-preprod.hmpps.service.justice.gov.uk
 
   env:
-#    SENTRY_ENV: preprod
+    # SENTRY_ENV: dev              # <== COMMENTED OUT to investigate failing build 20/12/2024
```

Comment:
A couple of comments dated Monday. Just checking that these are committed intentionally - if so, no problem.